src/cincinnati/mod.rs

```diff
@@ -55,6 +57,26 @@ lazy_static::lazy_static! {
         "Total number of errors while checking for updates.",
         &["kind"]
     ).unwrap();
+    static ref DEADEND_RELEASE_EXISTS : AtomicU8 = AtomicU8::new(DeadEndState::Unknown as u8);
+}
+
+/// For tracking a dead-end release.
+#[derive(Debug, Clone)]
+pub enum DeadEndState {
+    Unknown = 0,
+    True = 1,
+    False = 2,
+}
+
+impl From<Arc<AtomicU8>> for DeadEndState {
+    fn from(x: Arc<AtomicU8>) -> Self {
+        match x.load(Ordering::Relaxed) {
```

Comment:
Here and below, mind detailing why you went with `Relaxed` ordering?

Reply:
I think that was a bad choice